### PR TITLE
Fix argument errors

### PR DESCRIPTION
**Pull request by jbirdjavi**
https://rollbar.com/Cru/SummerMissions/items/2163/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,6 @@ sudo: false
 cache: bundler
 language: ruby
 rvm:
-  - 2.4.1
+  - 3.0.5
```
Comment (knutsenm) [inline .travis.yml:5]: Shall I migrate this to Actions?

**Review by knutsenm — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (jbirdjavi) [reply .travis.yml:5]: It'd probably be good to migrate it to Actions! :)

**Review by knutsenm — COMMENTED** [review]
Forgot to edit .ruby-version

**Comment by jbirdjavi** [comment]
Oops. Can you do that when you do that GHA migration?

**Pull request merged**